emissions.md

```diff
@@ -121,11 +121,30 @@ where:
 
 <br/>
 
-Non-CO<sub>2</sub> GHG emissions are proportional to the activity except for any reductions in emission intensity due to the MAC curve. As noted above, the MAC curves are assigned to a wide variety of technologies, mapped directly from [EPA 2019](#epa2019). Under a carbon policy, emissions are reduced by an amount determined by the MAC curve.
+
+`MAC`-related parameters:
+
+XML Tag | Description |
```

Comment:
You need a `|` at the very start of this line otherwise the header doesn't format correctly